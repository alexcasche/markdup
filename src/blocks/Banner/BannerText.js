import styled from 'styled-components'
import { shade, tint } from 'polished'

const BannerText = styled.p`
  color: ${props => shade(0.85, props.theme.colors.grey)};
  font-size: 13px;
  font-weight: 500;
  line-height: 20px;
  letter-spacing: 2px;
  text-transform: capitalize;
  span {
    margin-left: 5px;
    color: ${props => tint(0.95, props.theme.colors.grey)};
    font-weight: 400;
    letter-spacing: 1px;
  }
`

export default BannerText
