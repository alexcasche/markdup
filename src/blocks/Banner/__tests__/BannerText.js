import BannerText from '../BannerText'

describe('BannerText Element', () => {
  it('renders correctly', () => {
    const tree = render(
      <BannerText>
        Lines: <span>10</span>
        <br />
        Characters: <span>250</span>
      </BannerText>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
