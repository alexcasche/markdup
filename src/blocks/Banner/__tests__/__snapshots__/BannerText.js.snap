// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerText Element renders correctly 1`] = `
.c0 {
  color: #919398;
  font-size: 13px;
  font-weight: 500;
  line-height: 20px;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
  text-transform: capitalize;
}

.c0 span {
  margin-left: 5px;
  color: #afb1b6;
  font-weight: 400;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

<p
  className="c0"
>
  Lines: 
  <span>
    10
  </span>
  <br />
  Characters: 
  <span>
    250
  </span>
</p>
`;
